explorer/app/components/Project/components/Section/section.styles.ts

```diff
@@ -35,7 +35,7 @@ export const SectionContent = styled(Typography)`
   ${({ theme }) => theme.breakpoints.up(BREAKPOINT.TABLET)} {
     padding: 0;
   }
-`;
+` as typeof Typography;
```

Comment:
Required: see https://mui.com/material-ui/guides/typescript/#complications-with-the-component-prop